docs/design/values.md

```diff
@@ -97,13 +103,30 @@ These conversion steps combine to provide the transitive conversion table:
 |    to **reference** | direct init + materialize | ==        | materialize        |
 | to **initializing** | direct init               | copy init | ==                 |
 
-Reference expressions formed through temporary materialization are called
-[_ephemeral reference expressions_](#ephemeral-reference-expressions) and have
-restrictions on how they are used. In contrast, reference expressions that refer
-to declared storage are called
-[_durable reference expressions_](#durable-reference-expressions). Beyond the
-restrictions on what is valid, there is no distinction in their behavior or
-semantics.
+Reference expressions are divided into 2x2 sub-categories: they can be either
+[_ephemeral_](#ephemeral-reference-expressions) or
+[_durable_](#durable-reference-expressions), and either _owning_ or
+_non-owning_.
+
+Ephemeral reference expressions are formed through temporary materialization,
+and have restrictions on how they are used. In contrast, durable reference
+expressions refer to storage that outlives the expression, and typically has a
+declared name. Owning reference expressions can only refer to complete objects,
+whereas non-owning reference expressions can refer to both complete objects and
+sub-objects (such as class fields and base class sub-objects). As a consequence,
+only owning reference expressions can be destructively moved.
+
+Value binding and copy initialization can be applied to any reference
+expression, but materialization only produces owning ephemeral reference
+expressions. An owning reference expression can be implicitly converted to
+non-owning; this has no run-time effect because it merely discards static
+ownership information. Non-owning reference expressions can only be converted to
+owning reference expressions by round-tripping through copy-initialization and
+materialization. Non-durable-reference expressions cannot be implicitly
+converted to durable reference expressions at all.
+
+> **TODO:** Determine how these reference sub-categories relate to memory-safety
+> properties like uniqueness.
```

Comment:
Would be good to run the nomenclature here past the folks working on the safety story, so if nothing else they know they need to update it if they want to give a different meaning to "owning".